src/plugins/MatchStarter.ts

```diff
@@ -142,6 +142,9 @@ export class MatchStarter extends LobbyPlugin {
       if (count > 15) {
         this.lobby.DeferMessage('Match starts in 10 seconds', 'mp_start 10 sec', (count - 10) * 1000, true);
       }
+      if (35 < count) {
```

Comment:
While this is fine, you can invert this statement for consistency to line 142.
```suggestion
      if (count > 35) {
```